packages/core/src/components/fields/SchemaField.tsx

```diff
@@ -255,6 +255,14 @@ function SchemaFieldRender<T, F>(props: FieldProps<T, F>) {
   if (!hideError && errors && errors.length > 0) {
     classNames.push("field-error has-error has-danger");
   }
+  if (uiSchema?.classNames) {
+    if (process.env.NODE_ENV !== "production") {
+      console.warn(
+        "WARNING: 'uiSchema.classNames' is deprecated and may be removed in a major release; Use 'ui:classNames' instead."
```

Comment:
```suggestion
        "'uiSchema.classNames' is deprecated and may be removed in a major release; Use 'ui:classNames' instead."
```

Reply:
Will commit this before merging if no other feedback is provided, otherwise will update on another commit